### DISCUSSION TITLE
Pretrained weights

### DISCUSSION THREAD

**Discussion by callzhang** [ethvis/qd-track#52] [Q&A] [answered]
Do you have plan to release the pretrained weights? It will help us to faster validate the results and enable us to decide possible tracks to further research.

**Comment by callzhang** [comment, marked as answer]
Sorry I have found the pretrained weights. Can you provide code for a quick inference on the model?

**Reply by callzhang** [reply]
Inference code is found too.

**Reply by ghost** [reply]
where did you find the inference code?
